src/main/java/uk/gov/hmcts/ccd/domain/types/DocumentValidator.java

```diff
@@ -43,7 +61,7 @@ public BaseType getType() {
                                            final CaseFieldDefinition caseFieldDefinition) {
 
         // Empty text should still check against MIN - MIN may or may not be 0
-        if (isNullOrEmpty(dataValue)) {
+        if (Boolean.TRUE.equals(isNullOrEmpty(dataValue))) {
```

Comment:
```suggestion
        if (isNullOrEmpty(dataValue)) {
```
No need to change this